### PR TITLE
fix: display file name in overlay

### PR DESCRIPTION
**Pull request by githoniel**
- [X] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **docs update**
- [ ] This is a **metadata update**

### For Bugs and Features; did you add new tests?

since there is no test for overlayer yet, so I did not add test for it
### Motivation / Use-Case

we are not display filename in webpack5 for error now

bug

compare to error ovelay in webpack-dev-server V3

fixed

### Breaking Changes

no

### Additional Info

add filename for error

### CODE REVIEW

**Comment by alexander-akait** [comment]
Please update tests

**Comment by githoniel** [comment]
> Please update tests

should I add test for it or just fix the lint?

**Comment by snitin315** [comment]
> > Please update tests
> 
> should I add test for it or just fix the lint?

Both, CI should be all green.

File: client-src/overlay.js [hunk 1]
```diff
@@ -131,7 +131,8 @@ function show(messages, type) {
       const entryElement = document.createElement("div");
       const typeElement = document.createElement("span");
 
-      typeElement.innerText = type === "warnings" ? "Warning:" : "Error:";
+      typeElement.innerText =
+        (type === "warnings" ? "Warning:" : "Error:") + message.file || "";
```
Comment (snitin315) [inline client-src/overlay.js:135]: I think `message.file` is `undefined` for webpack 4. So we should only add it when it's defined.

Comment (githoniel) [reply client-src/overlay.js]: I have add comment for it.  file only work for typescript

**Review by snitin315 — COMMENTED** [review]
I think we should add a test case for this in https://github.com/webpack/webpack-dev-server/blob/master/test/e2e/overlay.test.js

**Comment by alexander-akait** [comment]
@snitin315 Yep, I think we can merge it, I will send test cases today, maybe we need more fields here

**Review by snitin315 — APPROVED** [review]

**Pull request merged**